cmake/app/boilerplate.cmake

```diff
@@ -308,6 +321,50 @@ foreach(root ${BOARD_ROOT})
       endif()
     endforeach()
   endif()
+
+  # Match the .conf files in the shield directories to make sure we are
+  # finding shields, e.g. wnc_m14a2a/wnc_m14a2a.conf.conf
+  file(GLOB_RECURSE shields_conf_refs_list
```

Comment:
I don't think we need this list, I believe that since we know which shields exist we can just check, for each enabled shield, if a corresponding .conf file exists, and if so add it to the list of shield_conf_files.